Priest.py

```diff
@@ -22,9 +23,7 @@ def launch_heal(self,target):
             target.current_hp = target.max_hp
 
     def heal_reduce_cooldown(self,fps):
-        self.heal_cooldown_remaining -= 1 / fps
-        if self.heal_cooldown_remaining < 0.001:
-            self.heal_cooldown_remaining = 0
+        self.heal_cooldown_remaining = test_utility.cooldown(self.heal_cooldown_remaining, fps)
```

Comment:
cooldown function could move from `test_utility` to `utility` since this function is used in production code instead of test code